lab-integration/src/main.ts

```diff
@@ -1,47 +1,80 @@
-import { DateTime } from "luxon";
 import { promises as fs, watch } from "fs";
+import * as lockfile from "proper-lockfile";
 import path from "path";
-import { TestsResult } from "./types";
+import { queue } from "async";
+import puppeteer from "puppeteer";
+import { DateTime } from "luxon";
+import { Machines, TestsResult } from "./types";
 
 const filePath = path.join(process.cwd(), "../test-results.json");
 
+const machines: Machines = {
```

Comment:
Let's use `object` and `machines[testResult.Id_da_Maquina as keyof typeof machines],` instead

Reply:
I think that we could use this.. but with the type things could be more clear, so I think we should keep it, unless you this approch could be better in another way that you could explain, no just for lessen code.

Reply:
Gotcha. Let's keep it.

Reply:
Hey, I think it's unnecessary to define `Machines` because we don't expect to use them elsewhere.

What if we do
```ts
const machines: { [key: string]: string } = {
  "5534": "1 - Prensa PRE-01",
  "5533": "2 - Prensa PRE-02",
  "5532": "3 - Prensa PRE-03",
};
```